hbase-server/src/main/java/org/apache/hadoop/hbase/master/cleaner/CleanerChore.java

```diff
@@ -485,10 +384,13 @@ public boolean setEnabled(final boolean enabled) {
   }
 
   /**
-   * Attemps to clean up a directory, its subdirectories, and files.
-   * Return value is true if everything was deleted. false on partial / total failures.
+   * Attemps to clean up a directory, its subdirectories, and files. Return value is true if
+   * everything was deleted. false on partial / total failures.
    */
-  private class CleanerTask extends RecursiveTask<Boolean> {
+  private final class CleanerTask extends RecursiveTask<Boolean> {
+
+    private static final long serialVersionUID = -5444212174088754172L;
```

Comment:
nit: do we really need this serialVersiolUID ?  I don't see any serialization...

Reply:
The RecursiveTask implements Serializable so we need a serialVersionUID, otherwise there will be a warning.